save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/CloneRepositoryController.kt

```diff
@@ -35,21 +38,23 @@ class CloneRepositoryController(
      * @return mono string
      */
     @PostMapping(value = ["/submitExecutionRequest"])
-    fun submitExecutionRequest(@RequestBody executionRequest: ExecutionRequest): Mono<ResponseEntity<*>>? {
+    fun submitExecutionRequest(@RequestBody executionRequest: ExecutionRequest): Response {
         val project = executionRequest.project
+        var projectId: String? = null
```

Comment:
You can declare it as a non-nullable val, and kotlin compiler allows it because this variable is assigned only once inside `try`